src/org/rascalmpl/interpreter/load/StandardLibraryContributor.java

```diff
@@ -54,7 +56,7 @@ public void contributePaths(List<ISourceLocation> l) {
 						}
 					}
 					else {
-						l.add(vf.sourceLocation("file","", path));
+						l.add(URIUtil.createFileLocation(path));
```

Comment:
I also reviewed all cases where we are building up file paths by hand. There weren't that many, so I fixed the one I did find.